File: Coding_Problems_Java/Find center of star graph.java
class Solution {
    public int findCenter(int[][] edges) {
        int n = edges.length;
        HashMap<Integer,Integer> hm = new HashMap<>();

        for(int[] edge: edges){
            int u = edge[0];
            int v = edge[1];

            hm.put(u, hm.getOrDefault(u, 0)+1);
            hm.put(v, hm.getOrDefault(v, 0)+1);
        }
        for (Map.Entry<Integer, Integer> entry : hm.entrySet()){
            if(entry.getValue() == n){
                return entry.getKey();
            }
        }
        return -1;
    }
}

File: Coding Problems - Python/Graphs/Find center of star graph.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        n = len(edges)
        hm = dict()

        for u, v in edges:
            hm[u] = hm.get(u, 0) + 1
            hm[v] = hm.get(v, 0) + 1
        
        for key, value in hm.items():
            if value == n:
                return key
